src/Storages/MergeTree/MergeTreeIndexVectorSimilarity.h

```diff
@@ -142,6 +141,7 @@ class MergeTreeIndexConditionVectorSimilarity final : public IMergeTreeIndexCond
 private:
     const VectorSimilarityCondition vector_similarity_condition;
     const unum::usearch::metric_kind_t metric_kind;
+    size_t expansion_search;
```

Comment:
```suggestion
    const size_t expansion_search;
```

Don't bother with this if you don't change anything else though, doesn't worth to run the whole CI just because of this.